packaging/sources/preupgrade-assistant-xml-logs.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/preupg/report_parser.py b/preupg/report_parser.py
+index 1a9047c..7223bf4 100644
+--- a/preupg/report_parser.py
++++ b/preupg/report_parser.py
+@@ -198,6 +198,25 @@ class ReportParser(object):
+ 
+         self.write_xml()
+ 
++    def fix_stderr(self):
++        """The function transform stdout to stderr."""
++        for rule in self.get_all_result_rules():
++        # Filter all check-import=stdout which are empty and remove them
++            for check in self.get_nodes(rule, "check"):
++                cim_stdout = [x for x in self.get_nodes(check, "check-import")
++                                if x.get('import-name') == "stdout"]
```

Comment:
It makes no sense to get a list of all the check-import=stdout, when later on you work just with the first element. However keep it as is as this is low priority and temporary patch which will be removed soon.